app/v2/template/template_schemas.py

```diff
@@ -68,12 +68,11 @@
 }
 
 
-def create_post_template_preview_response(template, body, url_root):
+def create_post_template_preview_response(template, body):
     return {
         "id": template.id,
         "type": template.template_type,
         "version": template.version,
         "body": body,
-        "subject": template.subject,
-        "uri": "{}v2/template/{}/preview".format(url_root, template.id)
+        "subject": template.subject
```

Comment:
It is possible for the subject to have placeholders. At the moment this will not replace the placeholders.